src/commands/bot-interfaces/get.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk'
+import {flags} from '@oclif/command'
+import makeDebug from 'debug'
+
+import * as BotInterfacesGetAPI from '../../mix/api/bot-interfaces'
+import * as MixFlags from '../../utils/flags'
+import MixCommand from '../../utils/base/mix-command'
+import {BotInterfacesGetParams, MixClient, MixResponse, MixResult} from '../../mix/types'
+import {DomainOption} from '../../utils/validations'
+
+const debug = makeDebug('mix:commands:bot-interfaces:get')
+
+export default class BotInterfacesGet extends MixCommand {
+  static description = `retrieve an interface for bot
+ 
+Use this command to get an interface for bot. The configuration ID
+can be retrieved using the bot-configs:list command.`
+
+  static examples = [
+    'Retrieve an interface for bot',
+    '$ mix bot-interfaces:get -B 32 - C 54',
+  ]
+
+  static flags = {
+    json: MixFlags.jsonFlag,
+    bot: MixFlags.botFlag,
+    config: MixFlags.appConfigurationFlag,
+    yaml: MixFlags.yamlFlag,
+  }
+
+  get domainOptions(): DomainOption[] {
+    debug('get domainOptions()')
+    return ['bot', 'config']
+  }
+
+  async buildRequestParameters(options: Partial<flags.Output>): Promise<BotInterfacesGetParams> {
+    debug('buildRequestParameters()')
+    const {bot: botId, config: configId} = options
+
+    return {botId, configId}
+  }
+
+  doRequest(client: MixClient, params: BotInterfacesGetParams): Promise<MixResponse> {
+    debug('doRequest()')
+    return BotInterfacesGetAPI.getBotInterfaces(client, params)
+  }
+
+  outputHumanReadable(_transformedData: any) {
```

Comment:
The _ in front of `transformedData` would indicate that the function argument is not used in the body of the function. Here, we are using it so remove the "_".